content/tutorial/nav.yml

```diff
@@ -1,111 +1,111 @@
           forceInternal: true
         - id: prerequisites
-          title: Prerequisites
+          title: आवश्यक शर्तें
           href: /tutorial/tutorial.html#prerequisites
           forceInternal: true
     - id: setup-for-the-tutorial
-      title: Setup for the Tutorial
+      title: ट्यूटोरियल के लिए व्यवस्था
       href: /tutorial/tutorial.html#setup-for-the-tutorial
       forceInternal: true
       subitems:
         - id: setup-option-1-write-code-in-the-browser
-          title: "Option 1: Write Code in the Browser"
+          title: "व्यवस्था विकल्प 1: ब्राउज़र में कोड लिखें"
           href: /tutorial/tutorial.html#setup-option-1-write-code-in-the-browser
           forceInternal: true
         - id: setup-option-2-local-development-environment
-          title: "Option 2: Local Development Environment"
+          title: "व्यवस्था विकल्प 1: स्थानीय विकास पर्यावरण"
           href: /tutorial/tutorial.html#setup-option-2-local-development-environment
           forceInternal: true
         - id: help-im-stuck
-          title: Help, I'm Stuck!
+          title: मदद, मैं अटक गया हूँ!
           href: /tutorial/tutorial.html#help-im-stuck
           forceInternal: true
     - id: overview
-      title: Overview
+      title: सिंहावलोकन
       href: /tutorial/tutorial.html#overview
       forceInternal: true
       subitems:
         - id: what-is-react
-          title: What Is React?
+          title: React क्या है?
           href: /tutorial/tutorial.html#what-is-react
           forceInternal: true
         - id: inspecting-the-starter-code
-          title: Inspecting the Starter Code
+          title: स्टार्टर कोड का निरीक्षण
           href: /tutorial/tutorial.html#inspecting-the-starter-code
           forceInternal: true
         - id: passing-data-through-props
-          title: Passing Data Through Props
+          title: Props के माध्यम से डेटा पास करना
           href: /tutorial/tutorial.html#passing-data-through-props
           forceInternal: true
         - id: making-an-interactive-component
-          title: Making an Interactive Component
+          title: इंटरएक्टिव कौम्पोनॅन्ट बनाना
           href: /tutorial/tutorial.html#making-an-interactive-component
           forceInternal: true
         - id: developer-tools
-          title: Developer Tools
+          title: डेवलपर टूल
           href: /tutorial/tutorial.html#developer-tools
           forceInternal: true
     - id: completing-the-game
-      title: Completing the Game
+      title: खेल को पूरा करना
       href: /tutorial/tutorial.html#completing-the-game
       forceInternal: true
       subitems:
         - id: lifting-state-up
-          title: Lifting State Up
+          title: लिफ्टिंग स्टेट अप
           href: /tutorial/tutorial.html#lifting-state-up
           forceInternal: true
         - id: why-immutability-is-important
-          title: Why Immutability Is Important
+          title: क्यों अपरिवर्तनीयता महत्वपूर्ण है
```

Comment:
अपरिवर्तनीयता might be too complex too

Reply:
not sure, too complex for someone to understand in my opinion

Reply:
Any other suggestions for this? What should we use?

Reply:
how does "अचल स्थिति hona kyu महत्वपूर्ण है" sound?

Reply:
i feel `अचल` is more complex than  `अपरिवर्तनीयता`
As `अपरिवर्तनीयता` itself says, `not changing`  and for `अचल` I had to google